src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,95 @@
+import * as Polymorphic from '@radix-ui/react-polymorphic'
+import React, { forwardRef } from 'react'
+import { CSS, StitchesVariants, styled } from 'stitches.config'
+import { Box } from '../Box'
+import { Button } from '../Button'
+import { ChevronLeft, ChevronRight } from '../Icons'
+
+const commonButton: CSS = {
+  width: '1em',
+}
+
+export interface PaginationProps {
+  /*
+    The total number of pages
+  */
+  count: number
```

Comment:
might not be that case anymore - but for comments oto show int he docs they had to be the double start form.

```
/**
 *
 */
```

Just check these show up in the storybook docs.